include/dali/core/common.h

```diff
@@ -60,7 +60,8 @@ enum class OpType {
   GPU = 0,
   CPU = 1,
   MIXED = 2,
-  COUNT = 3
+  COUNT = 3,
+  NONE = 4
```

Comment:
Is the behaviour of to_string(OpType), etc ok after this change?

Reply:
Fixed